dataland_analytics/analysis_scripts/scripts_with_api_read/check_for_dataset.py

```diff
@@ -0,0 +1,63 @@
+# Check for a list of companies if SFDR data sets exists for reporting periods 2022 and 2023
+
+import dataland_datasets
+import pandas as pd
+
+from dataland_analytics.config import *
+
+configuration = dataland_datasets.Configuration(
+    host=DATALAND_SERVER + "api", access_token=PERSONAL_USER_API_KEY
+)
+
+company_ids = [
+    "a3b02fec-d9db-4af8-8698-3b2c7ec6e396",
+    "61615c24-747b-431d-91b5-798a38f42a17",
+    "58a603dc-f53c-4499-add1-1c463b129aba",
+]
+
+with dataland_datasets.ApiClient(configuration) as api_client:
+    company_controller_api_instance = dataland_datasets.CompanyDataControllerApi(
+        api_client
+    )
+    list_of_dicts = []
+    for company_id in company_ids:
+        company_info = company_controller_api_instance.get_company_by_id(company_id)
+        list_of_dicts.append(
+            {
+                "company_name": company_info.company_information.company_name,
+                "LEI": company_info.company_information.identifiers["Lei"][0],
+                "company_id": company_id,
+            }
+        )
+
+df = pd.DataFrame.from_records(list_of_dicts)
+df.to_csv("list_of_companies.csv", index=False, header=True)
+
+no_data = []
+data_2022 = []
+data_2023 = []
+
+with dataland_datasets.ApiClient(configuration) as api_client:
+    sfdr_data_api_instance = dataland_datasets.SfdrDataControllerApi(api_client)
+    for dict in list_of_dicts:
```

Comment:
shadowing built-in keyword 'dict'; perhaps it makes also more sense to call it "company" or "companyData" here? this makes what follows more easily understandable